engine/storage/src/main/java/org/apache/cloudstack/storage/allocator/AbstractStoragePoolAllocator.java

```diff
@@ -95,7 +93,7 @@ public boolean configure(String name, Map<String, Object> params) throws Configu
             String globalStorageOverprovisioningFactor = configs.get("storage.overprovisioning.factor");
             storageOverprovisioningFactor = new BigDecimal(NumbersUtil.parseFloat(globalStorageOverprovisioningFactor, 2.0f));
             extraBytesPerVolume = 0;
-            String allocationAlgorithm = configs.get("vm.allocation.algorithm");
+            String allocationAlgorithm = VolumeOrchestrationService.VolumeAllocationAlgorithm.value();
```

Comment:
maybe rename to indicate it is only used for volumes? i.e. `volumeAllocationAlgorithm`

(on line 68, that is)